extensions/wikia/Search/classes/Result.php

```diff
@@ -64,78 +76,91 @@ public function getText( $field = 'text', $wordLimit = null ) {
 		$textAsString = static::limitTextLength( $textAsString, $wordLimit );
 
 		// if title and description both start with the same File: prefix, remove the prefix from description
-		$textAsString = $this->removePrefix($this->findFilePrefix($this->getTitle(false)), $textAsString);
+		$textAsString = $this->removePrefix( $this->findFilePrefix( $this->getTitle( false ) ), $textAsString );
+
 		return $textAsString;
 	}
 
 	/**
 	 * Get the hub name, translated to content language
+	 *
 	 * @return string
 	 */
 	public function getHub() {
-		return wfMessage('hub-'.$this->getText( 'hub_s' ))->text();
+		return wfMessage( 'hub-' . $this->getText( 'hub_s' ) )->text();
 	}
 
 	/**
 	 * Returns the string value of the document's title
+	 *
 	 * @see    WikiaSearchResultTest::testTitleFieldMethods
 	 * @return string
 	 */
-	public function getTitle($removeFilePrefix = true) {
+	public function getTitle( $removeFilePrefix = true ) {
 		$result = '';
-		if ( isset( $this->_fields[Utilities::field('title')] )  ) {
-			$result = $this->_fields[Utilities::field('title')];
+		if ( isset( $this->_fields[Utilities::field( 'title' )] ) ) {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "title" 8 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1192)